Modules/Relationships/test/Relationships.Domain.Tests/Tests/Aggregates/Relationships/DecomposeRelationshipTests.cs

```diff
@@ -90,4 +107,21 @@ public void Identity_to_can_only_decompose_once()
         // Assert
         acting.Should().Throw<DomainException>().WithError("error.platform.validation.relationshipRequest.relationshipAlreadyDecomposed");
     }
+
+    [Fact]
+    public void Two_identities_can_enter_into_a_new_relationship_again_after_decomposing()
+    {
+        // Arrange
+        var existingRelationships = new List<Relationship>
+        {
+            CreateDecomposedRelationship(IDENTITY_1, IDENTITY_2)
+        };
+
+        // Act
+        var newRelationship = new Relationship(RELATIONSHIP_TEMPLATE_OF_1, IDENTITY_2, DEVICE_2, null, existingRelationships);
+        newRelationship.Accept(IDENTITY_1, DEVICE_1, []);
```

Comment:
No need to accept. The important part is the creation.

Reply:
Removed the `Accept` part and changed the assertion to check for `RelationshipStatus.Pending`.